### PR TITLE
Blocks Tests

### PR DESCRIPTION
**Pull request by FarukOzderim**
- implement a simple test to check blocks config generation
#894

### CODE REVIEW

File: test/test_data/blocks_configs.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
+XRAY_CONFIG = {
+    "mode": "blocks",
+    "components": [
+        {
+            "id": 1,
+            "type": "markdown",
+            "props": {
+                "default_value": "<pre><code># Detect Disease From Scan\nWith this model you can lorem ipsum\n- ipsum 1\n- ipsum 2\n</code></pre>\n",
```
Comment (FarukOzderim) [inline test/test_data/blocks_configs.py:8]: This is the problematic part

File: test/test_data/blocks_configs.py [hunk 2]
```diff
@@ -1,149 +1,220 @@
...
-        {"targets": [9], "trigger": "click", "inputs": [2, 7], "outputs": [8]},
-        {"targets": [15], "trigger": "click", "inputs": [2, 13], "outputs": [14]},
-    ],
-}
+	'mode': 'blocks',
+	'components': [{
+		'id': 1,
+		'type': 'markdown',
+		'props': {
+			'default_value': '<pre><code>        # Detect Disease From Scan\n        With this model you can lorem ipsum\n        - ipsum 1\n        - ipsum 2\n</code></pre>\n',
```
Comment (FarukOzderim) [inline test/test_data/blocks_configs.py:7]: I got the config from from terminal with `print(xray_blocks.get_config_file())`, not sure why it does not comply with the test.
Thanks for the fix!

**Comment by FarukOzderim** [comment]
LGTM, feel free to merge if it is ok for you.

File: test/test_data/blocks_configs.py (hunk 2 above)
Comment (abidlabs) [reply test/test_data/blocks_configs.py:7]: Same lol

**Review by abidlabs — APPROVED** [review]

**Pull request merged**